### DISCUSSION TITLE
AM is mono, no BW button for mono needed

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#367] [Q&A]
Why do we need the BW button to switch to mono even on AM or OIRT? I don't understand the logic.

**Comment by PE5PVB** [comment]
On AM it is only a BW button (in latest commits). On OIRT sometimes MPX stereo is used.

**Reply by andimik** [reply]
I only could find "Bugfix: AUTO BW was tunable on AM", but this is about tuning.

**Comment by PE5PVB** [comment]
Yes, I think I forgot to add the fix of mono/stereo on AM to the commit descriptions.

**Comment by PE5PVB** [comment]
Sometimes I see some problems on the fly when doing other things on the software ;-)
